stories/index.js

```diff
@@ -106,6 +123,39 @@ storiesOf('Search Components/DataSearch', module)
     components: { BaseDataSearch },
     template: "<base-data-search :subProps=\"{ iconPosition: 'right', title: 'Book Store', showFilter: false}\"/>",
   }))
+  .add('without search icon', () => ({
+    components: { BaseDataSearch },
+    template: "<base-data-search :subProps=\"{ showIcon: false, showFilter: false}\"/>",
+  }))
+  .add('with show clear', () => ({
+    components: { BaseDataSearch },
+    template: "<base-data-search :subProps=\"{ showClear: true, clearIcon: X, showFilter: false}\"/>",
+  }))
+  .add('with debounce', () => ({
+    components: { BaseDataSearch },
+    template: "<base-data-search :subProps=\"{ debounce: 300, showFilter: false}\"/>",
+  }))
+  .add('With strictSelection', () => ({
+    components: { BaseDataSearch },
+    template: "<base-data-search :subProps=\"{ strictSelection: true, showFilter: false}\"/>",
+  }))
+  .add('Without autosuggest', () => ({
+    components: { BaseDataSearch },
+    template: "<base-data-search :subProps=\"{ autosuggest: false, showFilter: false}\"/>",
+  }))
+  .add('With fuzziness as a number', () => ({
+    components: { BaseDataSearch },
+    template: "<base-data-search :subProps=\"{ fuzziness: 1, showFilter: false}\"/>",
+  }))
+  .add('With fuzziness as AUTO', () => ({
+    components: { BaseDataSearch },
+    template: "<base-data-search :subProps=\"{ fuzziness: AUTO, showFilter: false}\"/>",
+  }))
+   .add('With highlight', () => ({
```

Comment:
For highlight to work, we need to add `v-html` to render raw HTML currently it is displaying something like this `<mark>Harry</mark>`

Reply:
@jyash97 yes I agree with that, I tried passing doing this, 

```js
   .add('With highlight', () => ({
    components: { BaseDataSearch },
    template: "<base-data-search rawHtml=\"<span style= 'color: red'>hey</span>\" 
    :subProps=\"{ highlight: true, showFilter: false}\"/>",
  }))
```

and doing this 

```js
  <DataSearch
    className="result-list-container"
    categoryField="authors.raw"
    componentId="BookSensor"
    :dataField="['original_title', 'original_title.search']"
    v-bind='subProps'
    v-on='subEvents'
    v-html='rawHtml'
  />
```

but this replaces `DataSearch` component with `hey`, so you let me know how exactly I can add `v-html`

Reply:
@jyash97 I don't think we need to manually inject v-html as a prop, I think it depends upon how the ```DataSearch ``` component is written in vue.js.
So, I guess @bietkul can help us in this.

Reply:
Just sync this branch with master once, it should be resolved.

Reply:
@bietkul but intially the branch has no conflicts? have you updated master?

Reply:
Just check it is resolved now.